packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/WfoSidebar.tsx

```diff
@@ -151,11 +153,31 @@ export const WfoSidebar: FC<WfoSidebarProps> = ({ overrideMenuItems }) => {
     ];
 
     return (
-        <EuiSideNav
-            mobileTitle={t('mobileTitle')}
-            toggleOpenOnMobile={toggleMobile}
-            isOpenOnMobile={isSideNavOpenOnMobile}
-            items={defaultMenu}
-        />
+        <div>
+            <EuiSideNav
+                mobileTitle={t('mobileTitle')}
+                toggleOpenOnMobile={toggleMobile}
+                isOpenOnMobile={isSideNavOpenOnMobile}
+                items={defaultMenu}
+            />
+            <div
+                style={{
```

Comment:
Styles should be in a styles file. Also we use the `css` prop instead of the `style` prop to pass styles

Reply:
I checked, there isn't such component. EUI are also using custom css`{{ position: 'fixed'}} `in such cases.

Reply:
Added the copyright component